src/components/Summary/AlterationSummary.vue

```diff
@@ -232,6 +238,23 @@ export default class AlterationSummary extends Mixins(
   /** Prop to perform validation. */
   @Prop() readonly validate: boolean
 
+  /** The future effective fee for Alteration. */
+  futureEffectiveFees: number = 0
+
+  async mounted () {
+    // fetches the fee prices to display in the text
+    const feePrices = await this.fetchFeePrices()
+    this.futureEffectiveFees = feePrices.futureEffectiveFees
+  }
+
+  /** Fetches the Fee prices to display in the text */
+  private async fetchFeePrices (): Promise<FeesIF> {
+    const result = await Promise.resolve(this.fetchFilingFees(FilingCodes.ALTERATION,
+      this.getEntityType, true))
+    if (!('filingFees' in result)) throw new Error('Failed to fetch fees prices')
```

Comment:
Is this the same as `!result.filingFees` or `!result['filingFees]`?

Reply:
Just to add a bit: if `result` object doesn't have a `filingFees` property, then `result.filingFees` is undefined (which is falsy).

Reply:
👍